3rdparty/python/user_reqs.lock

```diff
@@ -1166,16 +1134,6 @@
               "hash": "6be126e3225486dff286a8fb9a06246a5253f4c7c53b475ea5f5ac934e64194c",
               "url": "https://files.pythonhosted.org/packages/90/c7/6dc0a455d111f68ee43f27793971cf03fe29b6ef972042549db29eec39a2/psutil-5.9.8.tar.gz"
             },
-            {
-              "algorithm": "sha256",
-              "hash": "8cb6403ce6d8e047495a701dc7c5bd788add903f8986d523e3e20b98b733e421",
```

Comment:
@cburroughs do you have any insight into why this PR removed the manylinux wheels for psutil and cryptography from the lockfile? 

Do you recall how you regenerated the lockfile? I'm assuming `pants generate-lockfiles --resolve=python-default`?

Reply:
Narrowly, it is from `pip_version = "latest"` implying an upgrade to pip 25.1.

Noodling the consequences of that though.

Reply:
https://github.com/pex-tool/pex/issues/2753